server/ciphersuites_1.4.go

```diff
@@ -1,33 +0,0 @@
-// Copyright 2015 Apcera Inc. All rights reserved.
```

Comment:
So does this change now require at least 1.5 to build? If so, totally ok, but should update main README.